dev/integration_tests/ios_add2app_life_cycle/ios_add2appTests/IntegrationTests.m

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <EarlGrey/EarlGrey.h>
+#import <XCTest/XCTest.h>
+
+#import "../ios_add2app/AppDelegate.h"
+#import "../ios_add2app/FullScreenViewController.h"
+
+@interface FlutterTests : XCTestCase
+@end
+
+@implementation FlutterTests {
+  int _flutterWarmEngineTaps;
+}
+
+- (instancetype)init {
+  self = [super init];
+
+  if (self) {
+    _flutterWarmEngineTaps = 0;
+  }
+
+  return self;
+}
+
+- (void)expectSemanticsNotification:(UIViewController*)viewController
+                             engine:(FlutterEngine*)engine {
+   // Flutter app will only send semantics update if test passes in main.dart.
+  [self expectationForNotification:FlutterSemanticsUpdateNotification object:viewController handler:nil];
+  [self waitForExpectationsWithTimeout:30.0 handler:nil];
+}
+
+- (void)checkAppConnection {
+  FlutterEngine *engine;
+  __weak UIViewController *weakViewController;
+  @autoreleasepool {
```

Comment:
Were you seeing memory bloat here?  I would imagine the pool drains between tests but I don't know.